01-messaging.md

```diff
@@ -220,9 +221,16 @@ receiver to parse individual elements from `value`.
 Various fundamental types are referred to in the message specifications:
 
 * `byte`: an 8-bit byte
+* `i8`: an 8-bit signed integer
```

Comment:
nit: use `s` not `i`, as it's better to do opposites in specs (unsigned vs signed -> `u` vs `s`)

Reply:
Done in https://github.com/niftynei/lightning-rfc/pull/15/commits/cf62b5a2a205090b9bce9e7e9b164df5e98b79fa

I find it a bit weird, as `i32`/`i64` seems to be the standard way in programming languages to refer to signed integers and I've never seen `s32`/`s64` anywhere, but I don't really mind!

Reply:
Yeah `s` seems a bit strange but no big deal honestly.